test/integration/testutils.ts

```diff
@@ -37,18 +36,26 @@ export const clearCollections = async (dbUrl: string, collections: string[]) =>
 };
 
 export const cleanCollection = async (dbUrl: string, collection: string): Promise<any> => {
-	const conn = await mongo.connect(dbUrl);
+	const connection = await mongo.connect(dbUrl);
 	try {
-		await conn
+		const collections = await connection
 			.db('clinical')
-			.collection(collection)
-			.remove({});
-		await conn.db('clinical').dropCollection(collection);
```

Comment:
This block was the source of repeated errors that were just adding noise to the integration test runs
Errors of type `failed to drop collection ... ns not found`